darwin/exporter/formats/nifti.py

```diff
@@ -70,14 +83,20 @@ def export(
 
     """
 
-    if legacy:
+    if legacy is not None:
         console.print(
-            "Legacy flag is set to True. Annotations will be resized using legacy calculations.",
-            style="bold blue",
+            "The `legacy` flag is now non-functional and will be deprecated soon. The annotation import process now automatically detects if legacy annotation scaling is required.",
+            style="warning",
         )
 
     video_annotations = list(annotation_files)
     for video_annotation in video_annotations:
+        try:
+            medical_metadata = video_annotation.slots[0].metadata
+            legacy = not medical_metadata.get("handler") == "MONAI"  # type: ignore
```

Comment:
I think we also need to check if `is_volume` is true

Reply:
I actually don't think so. I tested, and files handles as slices return `"handler": null`

Reply:
Notes from our in-person conversation - These are the scenarios:

Before `MED_2D_VIEWER`:
- Not `MONAI`: Files are scaled --> It's possible to upload unscaled files, but these are files uploaded as slices (not volumes), which NifTI annotations cannot be exported from

After `MED_2D_VIEWER`:
- `MONAI`: Files are guaranteed to be unscaled
- Not `MONAI`: Files are scaled --> It's possible to upload unscaled files, but these are files uploaded as slices (not volumes), which NifTI annotations cannot be exported from